invenio_records_rest/utils.py

```diff
@@ -139,7 +166,7 @@ def data(self):
                 response = make_response(jsonify(data), data['status'])
                 response.headers['Location'] = location
                 abort(response)
-            except BuildError:
+            except (BuildError, ValueError):
```

Comment:
Do you have an example? Probably it should be check already in `build_pid_type_redirects` ...